script.js

```diff
@@ -0,0 +1,14 @@
+const img = document.querySelector('.union0');
+const menu = document.querySelector('.second-menu');
+const li = document.querySelectorAll('.second-menu li');
+const listItem = Array.from(li);
+const menuTgl = () => {
+  menu.classList.toggle('visibility');
+};
+img.addEventListener('click', menuTgl);
+listItem.forEach((child) => {
+  child.addEventListener('click', menuTgl);
+});
+document.getElementById('clickme').onclick = function click() {
+  this.classList.toggle('active');
+};
```

Comment:
**[REQUIRED CHANGES]**

- Hey, your implementation work but has some unexpected behaviour. Please see the video below and I'd try explain what you should do to fix it.

https://user-images.githubusercontent.com/59397159/173677105-b6f0dc0e-814c-4d9c-813e-55333d497165.mp4

- You can see from the video that after clicking on any link, the cancel button still appears instead of the hamburger icon. This behavior also flips around and vice versa. Please kindly ensure that the cancel button only shows when the hamburger icon is clicked and it disappears when any of the link is clicked.
- Also, kindly improve your designs to match the Figma design below:

*figma design*
![ok](https://user-images.githubusercontent.com/59397159/173678224-f5043f73-9d90-44a6-9964-c5b067bb9f13.png)

- The mobile menu should have a height that takes up the whole device height, that way the user must click on the link so as to view that section on your page. As of now, the user may chose to ignore the links and just scroll down the page which doesn't improve the user experience. Please, adjust your styling to match the design as much as you can.